okhttp/api/android/okhttp.api

```diff
@@ -1101,6 +1101,10 @@ public final class okhttp3/RequestBody$Companion {
 	public static synthetic fun create$default (Lokhttp3/RequestBody$Companion;[BLokhttp3/MediaType;IIILjava/lang/Object;)Lokhttp3/RequestBody;
 }
 
+public final class okhttp3/RequestBodyKt {
+	public static final fun sha256 (Lokhttp3/RequestBody;)Ljava/lang/String;
```

Comment:
Remind me, what's the policy for Java interop? Should I make this unaccessible from Java or use `@JvmName` so that it reads better?

Reply:
Just a regular polymorphic method please